src/main/java/pl/psnc/dei/ui/components/batches/BatchNavigationMenu.java

```diff
@@ -0,0 +1,63 @@
+package pl.psnc.dei.ui.components.batches;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.dependency.StyleSheet;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import pl.psnc.dei.ui.pages.BatchPage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+@StyleSheet("./styles/styles.css")
+public class BatchNavigationMenu extends VerticalLayout {
+
+    private BatchPage batchPage;
+    List<Button> elements = new ArrayList<>();
```

Comment:
Is there any particular reason for default access modifier of this list?

Reply:
There is not, I corrected this.